.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -4,21 +4,34 @@ about: Create a report to help us improve
 labels: bug
 ---
 
+<!--
+Checklist before submitting
+
+- [ ] Search through existing issue reports to check whether the issue already exists
+- [ ] Ensure you are running the latest software release, see https://github.com/linz/gazetteer/releases
+- [ ] Check QGIS/plugin log messages for more information.
+- [ ] If you can please create a light and self-contained sample dataset to reproduce the issue.
+-->
+
 ### Bug Description
-A clear and concise description of what the bug is and what you expected to happen.
+<!-- A clear and concise description of what the bug is and what you expected to happen. -->
 
 #### Steps to Reproduce
-Steps to reproduce the behavior:
+<!-- Steps to reproduce the behavior -->
+
 1. Go to '...'
```

Comment:
should this example also be commented out?

Reply:
Was thinking that bit was more like something the user would edit with their own information. So could leave it uncommented. I guess the principle is to minimise the amount of typing/deleting they need to do.

Reply:
sounds sound.